File: CS/TokenEdit_Glyph_InEndOfToken/CustomTokenEdit/TokenCheckedChangedEventArgs.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace TokenEdit_Glyph_InEndOfToken.CustomTokenEdit
{
    public class TokenCheckedChangedEventArgs : EventArgs
    {
        public TokenCheckedChangedEventArgs(CustomTokenEditToken _token)
        {
            Token = _token;
        }
         
        public CustomTokenEditToken Token { get; private set; }
    }
}

File: VB/CustomTokenEdit/TokenCheckedChangedEventArgs.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq

Namespace Example.CustomTokenEdit
	Public Class TokenCheckedChangedEventArgs
		Inherits EventArgs

		Public Sub New(ByVal _token As CustomTokenEditToken)
			Token = _token
		End Sub

		Private privateToken As CustomTokenEditToken
		Public Property Token() As CustomTokenEditToken
			Get
				Return privateToken
			End Get
			Private Set(ByVal value As CustomTokenEditToken)
				privateToken = value
			End Set
		End Property
	End Class
End Namespace
